src/ChatApp.jsx

```diff
@@ -229,28 +232,37 @@ export default function ChatbotUI() {
               }}
               id='textarea'
               placeholder="Type your message..."
-              className={` break-words outline-none w-full border-2 border-[#ffe7e1] shadow-[0px_4px_4px_0px_rgba(0,0,0,0.25)] h-[60px] m:h-[80px] rounded-[10px] bg-gray-800 text-white pl-[20px] py-[16px] m:py-[24px] pr-[50px] m:pr-[80px] font-light text-xl m:text-2xl resize-none overflow-y-auto `}
-              onInput={() => handleTextAreaResize(false)}
+              className={`${prompt==""?'':'cursor-pointer'} break-words outline-none px-4 w-full  h-[60px] m:h-[80px]  bg-gray-800 text-white pl-[20px] py-[16px] m:py-[24px] pr-[50px] m:pr-[80px] font-light text-md m:text-2xl resize-none overflow-y-auto `}
+              onInput={() => handleTextAreaResize(true)}
               onKeyDown={(e) => {
                 if (e.key === "Enter") {
                   e.preventDefault(); // Prevents newline on enter in the textarea
                   sendPrompt();
                   reset();
                 }
               }}
+              cols={1}
             />
+          </div>
+     
             <Button
               onClick={() => {
                 sendPrompt();
                 reset();
                 handleTextAreaResize(true)
               }}
               size="icon"
-              className={` absolute w-[30px] m:w-[40px] right-[20px] m:right-[20px] bottom-[26px] m:bottom-[35px] bg-transparent cursor-pointer hover:bg-gray-800`}
+              className={` self-end mb-4  bg-transparent cursor-pointer `}
             >
               <img src="/images/send.svg" alt="send" />
             </Button>
+        
+       
+         
+            </div>
+
           </div>
+          <p className="mb-5 text-center text-gray-400 mt-2 text-sm">VERSEwise Ai can make mistakes. Check important info.</p>
```

Comment:
Extract text to a resource file, as we might include localization in the future.
- [ ] Extract string values to a resource